include/xtl/xparallel_foreach.hpp

```diff
@@ -68,9 +68,9 @@ namespace xtl
             functor&& f
         ){
             auto workload = size;
-            const auto work_per_thread = pool.n_worker_threads();
+            const auto work_per_thread = workload /  pool.n_worker_threads();
```

Comment:
let's also be consistent with spaces around operators. **always** 1 space (`workload / pool.n...`).

Also, above is `double(work_per_thread)/double(XTL_DEFAULT_CHUNKING_FACTOR)` which should have spaces etc :)